app/src/lib/WalletModal.svelte

```diff
@@ -59,7 +58,7 @@
 			</button>
 
 			<ul class="wallet-adapter-modal-list">
-				{#each $walletConfigStore.wallets.slice(0, numberOfWalletsShown) as { name, icon }}
+				{#each $walletStore.wallets.slice(0, numberOfWalletsShown) as { name, icon }}
```

Comment:
@cogoo there is an issue here with the types. It seems that is changes in the new versions.

Reply:
Solved.